big-int/src/bigint.hpp

```diff
@@ -111,7 +100,7 @@ class largeInt
 
     //   POSTFIX OPERATORS
 
-    void operator=(largeInt);
+    void operator=(const largeInt &);
```

Comment:
Codacy found an issue: ['largeInt::operator=' should return 'largeInt &'.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285291)